### DISCUSSION TITLE
How is Photino different from MAUI?

### DISCUSSION THREAD

**Discussion by tombouderij** [tryphotino/photino.NET#125] [Q&A]
Microsoft has released MAUI, an open-source framework also for building native, cross-platform desktop applications with Web technology. What is the difference between Photino with Blazor UI and MAUI with Blazor UI?

**Comment by MikeYeager** [comment]
MAUI/Blazor hybrid apps seem to work very well sometimes and not so well others. Our experience has varied. They do have much better interop with JavaScript than Photino. The drawbacks to MAUI/Blazor hybrid are that it's a relatively large tech stack and configuration compared to Photino which is much more lightweight. Also, MAUI/Blazor doesn't run on Linux. At one point, we stopped work on Photino.Blazor because MAUI/Blazor should have displaced it, however it has yet to prove as small, fast and stable.

**Comment by Xyncgas** [comment]
You can develop webUI application using blazor but having the ability to do computing and IO and etc outside of browser, ON LINUX is enough a difference

Meanwhile for me, the application doesn't look or act different whether I run it in photino, or MAUI or simply open the application as a website from browser because I can set it up that way. It's a webUI, it's really a unified user experience
